docs/human-interface-guidelines.md

```diff
@@ -266,64 +266,64 @@ For more info, see the FreeDesktop.Org [menu entry](http://standards.freedesktop
+Make progress bars unambiguous by referring to a single, specific task. For example, use progress bars to indicate the status of lengthy processes like file transfers and encoding. Do not use progress bars to compound the progress of different types of action.
 
-* **Good Example**: Installation progress in Software Center
+* **Good Example**: Installation progress in AppCenter
 * **Bad Example**: Combined progress of downloading an album, burning a CD, and syncing a mobile device in Noise
 
 ### Badges {#badges}
 
-A badge shows a count of actionable items which your app manages. It's purpose is to inform the user that there are items that require user attention or action without being obtrusive. This is a passive notification. A badge should not show totals or rarely changing counters. If the badge is not easily dismissed when the user views your app, it is likely that this is not a good use of a badge.
+A badge shows a count of actionable items managed by your app. Its purpose is to inform the user that there are items that require user attention or action without being obtrusive. This is a passive notification. A badge should not show totals or rarely changing counters. If the badge is not easily dismissed when the user views your app, it is likely that this is not a good use of a badge.
 
-* **Good Example**: Unread messages in Geary
+* **Good Example**: Unread messages in Mail
 * **Bad Example**: Total number of Photos in Shotwell
 
 ## System Indicators {#system-indicators}
 
-Indicators are small icons that live on the top panel. They give users a place to glance for a quick indication of various settings or events. Clicking the icon shows a small menu with related actions available to the user.
+Indicators are small icons that live in the menu bar (wingpanel). They give users a place to glance for a quick indication of various settings or events. Clicking the icon shows a small menu with related actions available to the user.
 
 ![](/images/docs/human-interface-guidelines/system-indicators/systray.png)
 
-### Does Your App Need An Indicator? {#does-your-app-need-an-indicator}
+### Does Your App Need an Indicator? {#does-your-app-need-an-indicator}
 
-The indicator area is notorious for becoming cluttered and having inconsistent behavior. Given that users will probably install many third party apps, we need to be careful about how many indicators we're showing and how they behave. Keep in mind that not every application needs an indicator. Only a very small set of apps will benefit from one. You do not need an indicator if:
+The indicator area is notorious for clutter and inconsistent behavior. Given that users will probably install many third-party apps, we must be careful about the number of indicators we show and how they behave. Keep in mind that not every application needs an indicator. Only a very small set of apps will benefit from one. You do not need an indicator if:
```

Comment:
I think "becoming cluttered" is necessary because in its default state it is not cluttered. It only becomes cluttered when users install apps that overuse it.